deploy/crds/kabanero_v1alpha1_kabanero_crd.yaml

```diff
@@ -12,19 +12,6 @@ spec:
   scope: Namespaced
   subresources:
     status: {}
-  additionalPrinterColumns:
```

Comment:
Just curious why we're removing these extra fields that were just added....

Reply:
I am not sure about that, is that an auto-generate file? I don't recall editing it myself.